lib/msc_extensions/msc_unpublished_custom_refresh_token_lifetime/msc_unpublished_custom_refresh_token_lifetime.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:matrix/matrix.dart';
+
+extension MscUnpublishedCustomRefreshTokenLifetime on MatrixApi {
+  static const String customFieldKey = 'com.famedly.refresh_token_lifetime_ms';
```

Comment:
nit, maybe rename the variable to refreshTokenLifetime?

Reply:
Hmm, does this really have a benefit? That's already in the name if you take the full name. As it is a static variable the full name is:
`MscUnpublishedCustomRefreshTokenLifetime.customFieldKey`

Reply:
hm actually why is even a static variable then? It is only used in one place inside the extension itself. Can we just use the value instead of having it in a variable?

Reply:
Because this way you can fetch it if you need it. For example if you want to listen on changes you can filter the sync by this account data type